cpanspec

```diff
@@ -1295,11 +1298,10 @@ for my $ofile (@args) {
     }
     else {
         # Basic idea borrowed from Module::Depends.
-        my $intrusive = qq{perl $bin/bin/intrusive.pl $basedir$path};
-        my $jsondeps = qx{$intrusive};
-        my $deps = eval { $coder->decode($jsondeps) };
+        my $deps;
+        eval { $deps = Intrusive->new->dist_dir($basedir.$path)->find_modules; };
```

Comment:
I intentionally removed using `Intrusive` directly from `cpanspec` and put it into its own script so it can called in its own independent process.
Using Intrusive.pm will run Makefile.PL/Build.PL etc. and load other modules, so the script calling it should be as empty as possible otherwise. Using it from cpanspec directly means there are a ton of other modules already loaded, which can influence the behaviour.

However, the even better solution would be to just call Makefile.PL etc. directly via `qx` or `system` and then read the `MYMETA.*` files, but that is a bit more work.
If you have an idea how to package intrusive.pl so that it works for now, that would be nice.

Reply:
Done in #33